diagnostic.md

```diff
@@ -150,7 +172,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will float to the right and be positioned at the top of the right of the page. Divs two three and five will float left (they will be stacked vertically on top of each other to the left of the page). divFour will just be positioned normally between divs three and five because of the clearing.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5